pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-transformations-cli"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
Sorry for merging the other PR before this one, this needs one more version bump along with comment changes. :/